src/display.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable import/no-cycle */
+
+import {
+  addNewEvents,
+  deleteAllElements,
+  deleteListEvents,
+  dragDropEvents,
+  editListEvents,
+  listCompleteEvents,
+} from './elemevent.js';
+import List from './root.js';
```

Comment:
## WOW, GOOD JOB 🎊.👏 
You have added drag and  drop functionality correctly and its working 💯, GOOD LUCK in future projects 🎉🍀

Reply:
@Rustamxon7 Thank you!
<img src="https://media3.giphy.com/media/gEP2k49ndOqJDBSPZl/giphy.gif"/>